lib/text_parser_override.rb

```diff
@@ -0,0 +1,11 @@
+StartingBlocks::TextParser.class_eval do
+
+  def assertions
+    examples
+  end
+
+  def tests
+    examples
```

Comment:
I know, this is nutty... but I'm overriding ```method_missing``` to inspect the results of the text output for "X examples", or whatever.  I really enjoy Ruby.